lib/live_isolated_component.ex

```diff
@@ -163,4 +197,57 @@ defmodule LiveIsolatedComponent do
       )
     end
   end
+
+  @doc """
+  Asserts the return value of a handle_event
+  """
+  defmacro assert_return_handle_event_message(view, return_value) do
+    quote do
+      view_pid = unquote(view).pid
+
+      assert_receive {unquote(@handle_event_received_message_name), ^view_pid, _,
+                      unquote(return_value)}
+    end
+  end
+
+  @doc """
+  Asserts that a given handle event has been received.
+
+  Depending on the number of parameters, different parts are checked:
+
+  - With no parameters, just that a handle_event message has been received.
+  - With one parameter, just the event name is checked.
+  - With two parameters, both event name and the parameters are checked.
+
+  If you just want to check the parameters without checking the event name,
+  pass `nil` as the event name.
+  """
+  defmacro assert_receive_handle_event_message(view, event \\ nil, params \\ nil, socket \\ nil) do
+    event = if is_nil(event), do: quote(do: _event), else: event
+    params = if is_nil(params), do: quote(do: _params), else: params
+    socket = if is_nil(socket), do: quote(do: _socket), else: socket
+
+    quote do
+      view_pid = unquote(view).pid
+
+      assert_receive {unquote(@handle_event_received_message_name), ^view_pid,
+                      {unquote(event), unquote(params), unquote(socket)}, _}
+    end
+  end
```

Comment:
I think this assertion name can be shorter, as long as the function doc mentions that it has the same ergonomics as `assert_receive`.

I also think `timeout` should be an arg.

```suggestion
  defmacro assert_handle_event(view, event \\ nil, params \\ nil, socket \\ nil, timeout \\ 100) do
    event = if is_nil(event), do: quote(do: _event), else: event
    params = if is_nil(params), do: quote(do: _params), else: params
    socket = if is_nil(socket), do: quote(do: _socket), else: socket

    quote do
      view_pid = unquote(view).pid

      assert_receive {unquote(@handle_event_received_message_name), ^view_pid,
                      {unquote(event), unquote(params), unquote(socket)}, _}, unquote(timeout)
    end
  end
```